Cannot build on Termux

**V version:** 529abf5af73af1561c63a39769d0
**OS:** Termux

**What did you do?**

```
$ make
rm -f v.c v vprod
curl -Os https://raw.githubusercontent.com/vlang/vc/master/v.c
cc -std=gnu11 -w -o v v.c
/data/data/com.termux/files/usr/bin/arm-linux-androideabi-ld: /data/data/com.termux/files/usr/tmp/v-4669cd.o: in function `Table_cgen_name':
v.c:(.text+0xb540): undefined reference to `__atomic_load'
/data/data/com.termux/files/usr/bin/arm-linux-androideabi-ld: v.c:(.text+0xb578): undefined reference to `__atomic_load'
clang-8: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Makefile:5: v] Error 1
$ git log|head
commit 529abf5af73af1561c63a39769d0
Author: Alexander Medvednikov <alexander@medvednikov.com>
Date:   Mon Jul 1 02:58:49 2019 +0200

    fix http
```

--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@ all: clean v
 	$(info V has been successfully built)
 
 v: v.c
-	cc -std=gnu11 -w -o v v.c
+	cc -march=native -std=gnu11 -w -o v v.c
 	./v -o v compiler
 	rm v.c
 

--- a/compiler/main.v
+++ b/compiler/main.v
@@ -415,7 +415,7 @@ string _STR_TMP(const char *fmt, ...) {
 fn (v mut V) cc() {
 	linux_host := os.user_os() == 'linux'
 	v.log('cc() isprod=$v.pref.is_prod outname=$v.out_name')
-	mut a := ['-w']// arguments for the C compiler
+	mut a := ['-w', '-march=native']// arguments for the C compiler
 	flags := v.table.flags.join(' ')
 	/* 
 	mut shared := ''

